### DISCUSSION TITLE
Minor Bug when IP is changed through so-ip-update

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#7221] [Unsupported Versions] [answered]
I have noticed that after updating the IP address, in Elastic search, the _id Field that holds the Value "Hunt and optionally picot to PCAP/Cases" hold an URL that referce to the original IP address used at the time of the initial install. This value does not seem to get updated with the so-up-update tool.

Can someone confirm this is a Bug?

**Comment by mciantar** [comment, marked as answer]
Thanks!  Fixed in https://github.com/Security-Onion-Solutions/securityonion/issues/7237.

**Reply by dougburks** [reply]
Yes, that fix will be available in the upcoming 2.3.110 release. In the meantime, you can manually reload Kibana dashboards:
```
sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True
```

Also keep in mind that `so-ip-update` is an experimental utility:
https://docs.securityonion.net/en/2.3/ip.html

We typically recommend using a static IP address and not changing it:
https://docs.securityonion.net/en/2.3/configuration.html

**Reply by mciantar** [reply]
Thanks!
